### PR TITLE
refactor(foss): move all /sql queries

### PR DESCRIPTION
**Pull request by EDsCODE**
## Problem

- move all the relevant sql files to /posthog
## Changes

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: posthog/queries/util.py [hunk 1]
```diff
@@ -8,15 +8,18 @@
 from rest_framework.exceptions import ValidationError
 
 from posthog.client import sync_execute
-from posthog.models.event import DEFAULT_EARLIEST_TIME_DELTA
+from posthog.models.event.event import DEFAULT_EARLIEST_TIME_DELTA
```
Comment (neilkakkar) [inline posthog/queries/util.py:11]: was this change necessary, given the init file imports everything from `/event/event.py` ?

File: posthog/models/__init__.py [hunk 2]
```diff
@@ -5,10 +5,10 @@
 from .cohort import Cohort, CohortPeople
 from .dashboard import Dashboard
 from .dashboard_tile import DashboardTile
-from .element import Element
+from .element.element import Element
```
Comment (neilkakkar) [inline posthog/models/__init__.py:8]: same q

File: posthog/queries/trends/formula.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 from typing import Any, Dict, List
 
 from ee.clickhouse.queries.breakdown_props import get_breakdown_cohort_name
-from ee.clickhouse.sql.clickhouse import trim_quotes_expr
+from posthog.clickhouse.kafka_engine import trim_quotes_expr
```
Comment (neilkakkar) [inline posthog/queries/trends/formula.py:6]: Q: How did this util end up in `kafka_engine` ?

**Review by neilkakkar — APPROVED** [review]
minor comments

File: posthog/queries/trends/formula.py (hunk 3 above)
Comment (EDsCODE) [reply posthog/queries/trends/formula.py:6]: It was always there but the naming of the file was just "clickhouse" and I renamed it to be more accurate
